test.js

```diff
@@ -12,14 +12,32 @@ tape('unknown', function (t) {
 })
 
 tape('txt', function (t) {
-  testEncoder(t, packet.txt, Buffer.allocUnsafe(0))
-  testEncoder(t, packet.txt, Buffer.from('hello world'))
-  testEncoder(t, packet.txt, Buffer.from([0, 1, 2, 3, 4, 5]))
+  testEncoder(t, packet.txt, ['hello world'])
+  testEncoder(t, packet.txt, ['hello', 'world'])
+  testEncoder(t, packet.txt, [Buffer.from([0, 1, 2, 3, 4, 5])])
+  testEncoder(t, packet.txt, ['a', 'b', Buffer.from([0, 1, 2, 3, 4, 5])])
```

Comment:
Need a test for plain string.

Reply:
See the following tests for scalar string and buffer.